src/Symfony/Component/VarDumper/Caster/ConstStub.php

```diff
@@ -30,4 +30,27 @@ public function __toString()
     {
         return (string) $this->value;
     }
+
+    /**
+     * Creates ConstStub from bitfield flag.
+     *
+     * @param int    $value  The value of bitfield flag
+     * @param string $prefix The prefix that filters flags
+     *
+     * @return self
+     */
+    public static function fromFlag($value, $prefix)
+    {
+        $constants = get_defined_constants();
```

Comment:
the implementation currently supports only flags defined based on normal constants. Is it possible to support class constants as well (for other cases) ?